torchbox/wagtail-torchbox
# 1770 Add homepage hero images backend

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/views/4019870/pulses/1222851770)

### Description of Changes Made

- Add a new hero images (min 1, max 6) field to homepage. Note that this will need more FE work.
- Remove some unused homepage related models (HomePageHero and HomePageClient) to tidy up.

FE update:
1) Each time a user refreshes the homepage, they see a random hero image (new JS component adds `--visible` class to a random image from the list of hero images) 
2) Wagtail crop fill - desktop used in this case
3) Old unnecessary code tidied up
![image](https://github.com/torchbox/wagtail-torchbox/assets/51043550/f5cfbdba-edf8-44b0-be22-a742bd91a7cf)

### How to Test

- Go to homepage admin, e.g. http://localhost:8000/admin/pages/5/edit/ There should now be a new 'Hero images' field that allows up to 6 images.

FE update:
- Refresh the page several times to see random hero image loaded each time

### MR Checklist
